compare.py

```diff
@@ -25,7 +28,7 @@ def main():
 
     # here we have some simple example logic to compare predictions to figure out which
     # predictions are the best, so this varies from use-case to use-case
-    best_of_best = (None, None)
+    best_of_best = (None, None, None)
```

Comment:
I find this tuple just a tad confusing, one could use a [named tuple](https://docs.python.org/3/library/collections.html#namedtuple-factory-function-for-tuples-with-named-fields) or something like that to indicate what these fields are. But this is just nitpicking, one can figure it out when reading onwards.